Openshift/documentation/scripts/docDataExtracter.sh

```diff
@@ -62,6 +62,8 @@ ansible -i $INVENTORY_FILE_PATH OSEv3 -a 'systemctl is-active docker' > docker-i
 ansible -i $INVENTORY_FILE_PATH OSEv3 -a 'lsblk' > lsblk.log
 ansible -i $INVENTORY_FILE_PATH OSEv3 -a 'cat /etc/ntp.conf' > ntp.log
 ansible -i $INVENTORY_FILE_PATH OSEv3 -a 'ifconfig' > ifconfig.log
+ansible -i $INVENTORY_FILE_PATH masters -a 'cat /etc/fstab' > fstab_masters.log
```

Comment:
lets get data of fstab from all nodes instead of just master.

Reply:
Agree will change.